static/js/overlays.js

```diff
@@ -134,6 +134,8 @@ exports.open_modal = function (selector) {
 };
 
 exports.close_overlay = function (name) {
+    popovers.hide_all();
```

Comment:
This feels like it should be done in another commit.

Reply:
Agreed. I have made separate commit and also explained the need to add this in the commit message clearly. Thanks.